projects/dotcms-field-elements/src/components/dot-textfield/dot-textfield.tsx

```diff
@@ -47,10 +57,22 @@ export class DotTextfieldComponent {
     }
 
     componentWillLoad(): void {
+        this.fieldAttr.name = this.name;
+        this.validateProps();
         this.status = getOriginalStatus(this.isValid());
         this.emitStatusChange();
     }
 
+    @Watch('regexCheck')
+    regexCheckWatch(): void {
+        this.regexCheck =
+            dotPropValidator({
+                field: { ...this.fieldAttr },
```

Comment:
See comments in https://github.com/dotCMS/core-web/pull/1021/files#diff-15356a60e5d00893978145ac20b662e5